cpp/include/cudf_test/column_wrapper.hpp

```diff
@@ -577,6 +603,81 @@ class fixed_point_column_wrapper : public detail::column_wrapper {
                                    detail::make_null_mask(v, v + size),
                                    cudf::UNKNOWN_NULL_COUNT});
   }
+
+  /**
+   * @brief Construct a nullable column from a list of decimal elements using another list to
```

Comment:
Maybe use "initializer list" instead of "list" here.   "list" is kind of an overloaded term in this module (lists columns)

Reply:
Done.